### PR TITLE
Fix memory allocation bug in msgpack reader

### PR DESCRIPTION
**Pull request by acmiyaguchi**
This fixes a bug with the `test-cli-integration` script in Linux by allocating more memory to the msgpack unpacker before reading data into libprio data-structures.

### CODE REVIEW

File: libprio_wrap.c [hunk 1]
```diff
@@ -3196,11 +3196,19 @@ PyObject* PrioTotalShare_write_wrapper(const_PrioTotalShare p) {
 SECStatus PrioPacketVerify1_read_wrapper(PrioPacketVerify1 p, const unsigned char *data, unsigned int len, const_PrioConfig cfg) {
     SECStatus rv = SECFailure;
     msgpack_unpacker upk;
-    bool result = msgpack_unpacker_init(&upk, len+1);
+    // Initialize the unpacker with a reasonably sized initial buffer. The
+    // unpacker will eat into the initially reserved space for counting and may
+    // need to be reallocated.
+    bool result = msgpack_unpacker_init(&upk, 128);
```
Comment (wlach) [inline libprio_wrap.c]: I guess this file is autogenerated based on the `.i`?

File: scripts/test-cli-integration [hunk 2]
```diff
@@ -37,7 +38,7 @@ EOF
 
 jq -c '.' ${CLIENT_BUCKET}/data.ndjson
 
-python -m prio encode-shares \
+python3 -m prio encode-shares \
```
Comment (wlach) [inline scripts/test-cli-integration]: Is the python3 command necessary?

File: libprio_wrap.c (hunk 1 above)
Comment (acmiyaguchi) [reply libprio_wrap.c]: Yes, I've found it useful to include the diff of the generated file.

File: scripts/test-cli-integration (hunk 2 above)
Comment (acmiyaguchi) [reply scripts/test-cli-integration]: Yes, unfortunately python3 is not the default python in the Fedora or Ubuntu images.

**Pull request merged**